packages/docs/stories/templates/appshell.stories.tsx

```diff
@@ -0,0 +1,502 @@
+import React from 'react';
+import type { Meta, StoryObj as Story } from '@storybook/web-components';
+import {
+  Description,
+  Stories,
+  Subtitle,
+  Title,
+} from '@storybook/blocks';
+import { html } from 'lit';
+import storyBookPreviewConfig from '../../.storybook/preview.js';
+import { generateStoryDescription } from '../../src/helpers/component.js';
+import { getTranslation } from '../../src/translations.js';
+import { generateFigmaPluginObject } from '../../src/helpers/figma.js';
+
+/**
+ * Create shared styles for all stories
+ */
+const createSharedStyles = () => html`
+  <style>
+    body {
+      margin: 0 !important;
+      padding: 0 !important;
+    }
+
+    syn-side-nav {
+      --side-nav-open-width: 320px;
+    }
+
+    #storybook-root,
+    #root-inner {
+      height: 100%;
+    }
+
+    .synergy-demo-application {
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      min-height: 550px;
+    }
+
+    .synergy-demo-content {
+      background: var(--syn-color-neutral-100);
+      display: flex;
+      flex: 1 0 auto;
+      flex-direction: column;
+      height: 1%;
+      position: relative;
+    }
+
+    .synergy-demo-content-inner {
+      display: flex;
+      flex: 1 0 auto;
+      flex-direction: column;
+    }
+
+    .synergy-demo-main {
+      align-items: center;
+      border: 1px dashed #9747FF;
+      border-radius: var(--syn-border-radius-medium);
+      color: #9747FF;
+      display: flex;
+      flex: 1 0 auto;
+      flex-direction: column;
+      font: var(--syn-body-small-bold);
+      justify-content: center;
+      margin: var(--syn-spacing-2x-large) var(--syn-spacing-2x-large) 0;
+    }
+
+    .synergy-demo-main h1 {
+      color: var(--syn-typography-color-text);
+      font: var(--syn-heading-large);
+      margin: 0;
+    }
+
+    .synergy-demo-footer > nav {
+      align-items: center;
+      display: flex;
+      font: var(--syn-body-x-small-regular);
+      gap: var(--syn-spacing-large);
+      height: var(--syn-spacing-3x-large);
+      justify-content: center;
+    }
+
+    .synergy-demo-footer :is(a, strong) {
+      color: var(--syn-input-help-text-color);
+    }
+  </style>
+`;
+
+/**
+ * Create the defaut syn-prio-nav for all stories
+ */
+const createPrioNav = () => html`
+  <!-- prio-nav -->
+  <syn-prio-nav slot="navigation">
+    <syn-nav-item current>${getTranslation('appShell.navigation.home')}</syn-nav-item>
+    <syn-nav-item>${getTranslation('appShell.navigation.documents')}</syn-nav-item>
+    <syn-nav-item>${getTranslation('appShell.navigation.applications')}</syn-nav-item>
+    <syn-nav-item>${getTranslation('appShell.navigation.teams')}</syn-nav-item>
+  </syn-prio-nav>
+  <!-- /prio-nav -->
+`;
+
+/**
+ * Create a header with optional prio-nav
+ */
+const createHeader = ({
+  withMetaNavigation = false,
+} = {}) => html`
+  <!-- header -->
+  <syn-header>
+  ${getTranslation('appShell.appName')}
```

Comment:
When using the label it will still fail as there is other content in the default slot (which is the reason for the whole 2.x release anyways ;)). I can only fix this when 2.x is available. If we get the template to be merged before the 2.x release it should not be that much of a problem as we can adjust before 2.x. If it comes after, we will have to merge it later

Reply:
Then I´m fine with it. We just need to keep in mind to change it as soon as we merge the 2x. branch